passage/src/androidTest/java/id/passage/android/TokenStoreTests.kt

```diff
@@ -111,4 +116,57 @@ internal class TokenStoreTests {
                 assertThat(e is PassageUserUnauthorizedException)
             }
         }
+
+    @Test
+    fun getValidAuthTokenWithValidToken() =
+        runTest {
+            val result = passage.tokenStore.getValidAuthToken()
+            assertNotNull(result)
+        }
+
+    @Test
+    fun getValidAuthTokenWithInvalidToken() =
+        runTest {
+            try {
+                passage.tokenStore.setTokens(AuthResult("invalid", ""))
+                passage.tokenStore.getValidAuthToken()
+                fail("Test should throw PassageTokenException")
+            } catch (e: Exception) {
+                assertThat(e is PassageTokenException)
+            }
+        }
+
+    @Test
+    fun isAuthTokenValidWithValidToken() {
+        val validToken = IntegrationTestConfig.AUTH_TOEKN
+        val result = passage.tokenStore.isAuthTokenValid(validToken)
+        assertThat(result).isTrue()
+    }
+
+    @Test
+    fun isAuthTokenValidWithInvalidToken() {
+        val invalidToken = "invalidAuthToken"
+        val result = passage.tokenStore.isAuthTokenValid(invalidToken)
+        assertThat(result).isFalse()
+    }
+
+    @Test
+    fun revokedWithValidToken() =
+        runTest {
+            val validRefreshToken = refreshToken
+            passage.tokenStore.revokeRefreshToken(validRefreshToken)
+            // should not throw any error
+        }
+
+    @Test
+    fun refreshWithInvalidToken(): Unit =
+        runBlocking {
+            try {
+                val invalidRefreshToken = "invalid"
+                val authResult = passage.tokenStore.refreshAuthToken(invalidRefreshToken)
+                fail("Test should throw PassageTokenException")
+            } catch (e: Exception) {
+                assertThat(e is PassageTokenException)
+            }
+        }
```

Comment:
Based on our conversations in Slack, we can remove this test as it will not pass (which is expected).
```suggestion
```

Reply:
this one works, only the revoke function with invalid token doesn't work